cmd/prometheus/uname_linux_int8.go

```diff
@@ -16,10 +16,12 @@
 
 package main
 
+import "strings"
+
 func charsToString(ca []int8) string {
 	s := make([]byte, len(ca))
 	for i, c := range ca {
 		s[i] = byte(c)
```

Comment:
Can you break out of the loop if the byte is a null byte? Just print out the `int8` value and check against that.